airflow/models/taskinstance.py

```diff
@@ -376,7 +379,8 @@ def log_url(self):
             "execution_date={iso}"
             "&task_id={task_id}"
             "&dag_id={dag_id}"
-        ).format(iso=iso, task_id=self.task_id, dag_id=self.dag_id)
+            "&try_number={try_number}"
```

Comment:
This actually will not work. `try_number` is not accepted by `/log` endpoint as an argument. 
Please refer to https://github.com/apache/airflow/blob/master/airflow/www/views.py#L581.